### ISSUE TITLE
Improve emojis accessibility 

### ISSUE THREAD

**Issue by LauraBeatris** [LauraBeatris/floripamais#20]
## Description 

Copying the emojis by character doesn't work on all systems and we still need to care about accessibility. 

The emojis don't work equally on all operating systems because of the Unicode difference. Windows 7 was released in 2009 - 3 years before the first emoji was invented, which was added to Unicode 6.1 in 2012.  

I don't think it's a good idea to replace the emojis, which are just Unicode symbols, with images because it'll have to be loaded unnecessarily. The choice to use an old operating system has its consequences.

With that being said, let's focus on the accessibility of the devices/operating system that support emojis. There's a lib https://github.com/seanmcp/a11y-react-emoji that handles the accessibility implementations related to emojis for us. 

## Tasks

- [x] Apply https://github.com/seanmcp/a11y-react-emoji
